### ISSUE TITLE
[Doc] Search doesn't work on pulsar website

### ISSUE THREAD

**Issue by ocadaruma** [apache/pulsar#22883]
### Search before asking

- [X] I searched in the [issues](https://github.com/apache/pulsar/issues) and found nothing similar.

### What issue do you find in Pulsar docs?

As of 2024-06-09, search bar of `https://pulsar.apache.org/` doesn't show any result.

<img width="1224" alt="image" src="https://github.com/apache/pulsar/assets/6070684/2c7df22f-e9bf-4ce4-b172-98eeb749c57f">

On the other hand, the search does work for `https://pulsar.apache.org/docs/3.2.x/`. Latest version is not indexed in Algolia somehow?

### What is your suggestion?

Fix search functionality to work

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by lhotari** [comment]
This seems to be fixed now. 

It was probably caused when 3.3.x version was added. The Pulsar Site seems to require a full rebuild in that case. Yesterday I did the full rebuild by committing an empty commit with a specific commit message to trigger the rebuild.

```
git commit -m 'Rebuild: BUILD_ALL_VERSION=1' --allow-empty
```
the build took about 1h 20mins: https://github.com/apache/pulsar-site/actions/runs/9454073305/job/26040821948
